src/index.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from "@ledgerhq/devices";
 import type {DeviceModel} from "@ledgerhq/devices";
 import {log} from "@ledgerhq/logs";
-import {Observable, defer, merge, from, Subject} from "rxjs";
+import {Observable, defer, merge, from, Subject, firstValueFrom} from "rxjs";
```

Comment:
I had some issues(forgot exactly what it was), when upgrading rxjs to v7. But I guess if tests are succeeding, I guess it's fine for now.